src/domain/safe/entities/multisig-transaction.entity.ts

```diff
@@ -10,7 +8,7 @@ export interface Confirmation {
   signature?: string;
 }
 
-export interface MultisigTransaction {
+export type MultisigTransaction = {
```

Comment:
Not strictly related to the PR, but could be a good point to change this?
```
dataDecoded?: any;
```
vs
```
dataDecoded?: DataDecoded;
```

Reply:
I would tackle it in a different PR. There were already quite some schemas introduced and I didn't want to add a new one here.